cypress/e2e/firstTests.cy.js

```diff
@@ -0,0 +1,23 @@
+/// <reference types="cypress" />
+
+describe('', () => {
+  it('', () => {
+    cy.visit ('https://react-redux.realworld.io/#/login?_k=ks2hqr')
```

Comment:
It's better not to place space between method and parentheses
```suggestion
    cy.visit('https://react-redux.realworld.io/#/login?_k=ks2hqr')
```